miso-web/src/main/webapp/pages/viewStats.jsp

```diff
@@ -53,7 +53,6 @@
           <th>Health</th>
           <th>Start Date</th>
           <th>Last Updated</th>
-          <th class="fit">View Stats</th>
```

Comment:
I think the stuff removed here is something EI uses (statsdb configurable in miso.properties). Check with @frankie13 / @froggleston before removing